src/assets/javascripts/angular/services/academicsService.js

```diff
@@ -183,6 +183,43 @@ angular.module('calcentral.services').service('academicsService', function() {
     return classes;
   };
 
+  var getSwapClasses = function(enrolledCourses, waitlistedCourses) {
+    var count = 0;
+    var swapClassList = [];
+    // initialize swap_number
+    _.forEach(enrolledCourses, function(course) {
+      course.swap_number = 0;
+    });
+    _.forEach(waitlistedCourses, function(course) {
+      course.swap_number = 0;
+    });
+    // the following code matches the waitlist's swap with the enrolled class
+    for (var i = 0; i < enrolledCourses.length; i++) {
```

Comment:
@frankiepun - I intended to point out that it would read easier if you use the `_forEach()` on any `for` loops in the JavaScript code, including these.